packages/server/postgres/migrations/1614900793895_addTeamTable.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`
+    CREATE TYPE "MeetingTypeEnum" AS ENUM (
+      'action',
+      'retrospective',
+      'poker'
+    );
+
+    CREATE TABLE "Team" (
+      id VARCHAR(100) PRIMARY KEY,
+      name VARCHAR(100) NOT NULL,
+      "createdAt" TIMESTAMP NOT NULL,
+      "createdBy" VARCHAR(100) NOT NULL,
```

Comment:
there are many orphaned teams in rethink. i.e. teams whose "createdBy" points to a nonexistent user in rethink. do we want to keep these records? i've taken the foreign key constraint off `createdBy` for now. the alternative is to keep the constraint but skip backfilling the orphaned records.

Reply:
hmmm i'll add the FK constraint later. i think the first pass should just be mirroring what we have in rethink. don't wanna get too bogged down with adding the extra data integrity when what we really need is the performance.